docs/developer-docs/integrations/sns/tokenomics/sns-checklist.md

```diff
@@ -139,7 +122,7 @@ It’s strongly recommended that developers have a discussion in the forum with
 
 It is recommended to share:
 
-- [x] The init file and swap file (example: [OpenChat](https://github.com/open-ic/open-chat/tree/master/sns/config)). <!-- NEW: init file-->
```

Comment:
What actually is the swap file?

Reply:
I don't know :D 
I assume this meant to say the information that you put into the openSwap proposal, but I am not aware of any file

Reply:
will clarify and remove this part